docs/guide/choose-azure-container-service.md

```diff
@@ -116,7 +123,7 @@ AKS provides the most configurability, but this configurability requires more op
 
 ### Rule of thumb
 
-Generally, services that offer more simplicity tend to suit customers who prefer to focus more on feature development and less on infrastructure. Services that offer more control tend to suit customers who need more configurability and have the skills, resources, and business justification necessary to manage their own infrastructure.
+Generally, services that offer more simplicity tend to suit customers who prefer to focus on feature development rather than infrastructure management. Services that offer more control tend to suit customers who need more configurability and have the skills, resources, and business justification necessary to manage their own infrastructure. AKS Automatic provides a balanced approach, reducing operational overhead while offering more control than fully managed PaaS services like Azure Container Apps and App Service. Standard AKS offer the most control and are suited for customers with advanced Kubernetes expertise and the need for extensive configurability.
```

Comment:
Can we change "reducing operational **overhead**" to "reducing operational **effort**"?